WeakAurasOptions/LoadOptions.lua

```diff
@@ -461,7 +462,7 @@ function OptionsPrivate.ConstructOptions(prototype, data, startorder, triggernum
           options[name..suffix] = {
             type = "input",
             width = arg.noOperator and WeakAuras.normalWidth or WeakAuras.halfWidth,
-            validate = ValidateNumeric,
+            validate = (name == "remaining" and ValidateNumericOrPercent) or ValidateNumeric,
```

Comment:
you are changing all inputs of type number here, without any regard to whether they should or shouldn't support percents